### PR TITLE
Addition of news websites

### PR DESCRIPTION
**Pull request by sauhardjain**

### CODE REVIEW

File: websites_list/news_websites.csv [hunk 1]
```diff
@@ -1988,3 +1988,13 @@
 1986,New Indian Express,newindianexpress.com,Right Leaning
 1987,Dainik Bhaskar,bhaskar.com,Pro-NDA
 1988,Dainik Jagran,jagran.com,Pro-NDA
+1989,moneycontrol,moneycontrol.com,Ecnonomic news. Owned by TV18
```
Comment (danim3014) [inline websites_list/news_websites.csv]: control money, don't let money control you

File: websites_list/news_websites.csv [hunk 2]
```diff
@@ -1988,3 +1988,13 @@
...
+1990,Bleacher Report,bleacherreport.com
+1991,Eenadu,eenadu.net,Telugu News
+1992,Great Andhra,greatandhra.com,Regional news on Andhra Pradesh
+1993,Dinamalar,dinamalar.com,National Tamil Daily
+1994,Web Dunia,hindi.webdunia.com,Indpendent News Website focused on news in Hindi
+1995,One India,oneindia.com,Leaning right. Owned by BG Mahesh
+1996,Manorama Online,manoramaonline.com,Malyalam newspaper. Generally reliable
+1997,Rediff News,rediff.com/news,Founded by Ajit Balakrishnan
+
```
Comment (anonymous2308) [inline websites_list/news_websites.csv]: 1998,EFlip News,www.eflip.com/tag/News/,Underrated news website

Comment (anonymous2308) [inline websites_list/news_websites.csv]: 1999,The NewsHook,thenewshook.com/,Helps in learning the news

Comment (taylorylee) [inline websites_list/news_websites.csv]: 2000,Crikey inq, https://www.crikey.com.au/, On politics, media, business, the environment and life

Comment (taylorylee) [reply websites_list/news_websites.csv]: 2001,India Today, https://www.indiatoday.in/india, latest India news & top news on India Today.

Comment (taylorylee) [reply websites_list/news_websites.csv]: I made more changes https://github.com/AOSSIE-Org/Social-Street-Smart/compare/master...taylorylee:patch-1

File: websites_list/news_websites.csv (hunk 1 above)
Comment (shuhui00) [reply websites_list/news_websites.csv]: 1821, Guardian, https://www.theguardian.com/international, Liberal Democrats

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1964, The Sun, https://www.thesun.co.uk/, Conservative Party

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1999, http://www.taipeitimes.com/, owned by The Liberty Times Group

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1971, The Star, https://www.thestar.com.my/, left leaning

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1999, Malaysiakini, https://www.malaysiakini.com/, claims to be pro-government

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1980, Liberty Times, https://www.ltn.com.tw/, centre-left

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1951, United Daily News, http://udn.com/NEWS/, centre-right

Comment (shuhui00) [reply websites_list/news_websites.csv]: 2003, Apple Daily, https://tw.appledaily.com/, liberal

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1950, China Times, https://www.chinatimes.com/, centre-right

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1949, Taiwan News, https://www.taiwannews.com.tw/, centre-left

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1952, The China Post, https://chinapost.nownews.com/, centre-right

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1978, The Borneo Post, https://www.theborneopost.com/, daily english news in Sarawak and Sabah

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1976, Business Times, https://www.businesstimes.com.sg/, Owned by Singapore Press Holdings

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1994, The Edge, https://www.theedgemarkets.com/categories/malaysia, owned by The Edge Communications Sdn Bhd

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1949, Frankfurter Allgemeine Zeitung, https://www.faz.net/aktuell/, centre-right

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1854, Whitby Gazette, https://www.whitbygazette.co.uk/, impartial

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1996, Scottish Socialist Voice, https://socialistvoice.scot/, left leaning

Comment (shuhui00) [reply websites_list/news_websites.csv]: 2009, The Tab, https://thetab.com/, left leaning

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1845, New Straits Times, https://www.nst.com.my/, Owned by Media Prima

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1946, China Press, https://www.chinapress.com.my/, owned by The China Press Berhad

Comment (shuhui00) [reply websites_list/news_websites.csv]: 2006, Sinar Harian, https://www.sinarharian.com.my/, Malay language daily newspaper

Comment (shuhui00) [reply websites_list/news_websites.csv]: 2007, L'essentiel, www.lessentiel.lu/fr/, free daily newspaper

**Comment by tomd-04** [comment]
2008, Daily Express (UK), https://www.express.co.uk/, English Tabloid owned by Reach Plc. Right leaning.

**Comment by tomd-04** [comment]
2008, Daily Express (UK), https://www.express.co.uk/, English Tabloid owned by Reach Plc., Right leaning.

File: websites_list/news_websites.csv (hunk 1 above)
Comment (shuhui00) [reply websites_list/news_websites.csv]: 2000, Today, https://www.todayonline.com/, free daily newspaper

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1998, National Post, https://nationalpost.com/, liberal

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1980, Calgary Sun, https://calgarysun.com/, conservative

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1905, Lethbridge Herald, https://lethbridgeherald.com/, centre-left

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1936, The Globe and Mail, https://www.theglobeandmail.com/, centre-right

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1882, The Daily Courier https://www.dailycourier.com/, liberal

Comment (shuhui00) [reply websites_list/news_websites.csv]: 1903, Edmonton Journal, https://edmontonjournal.com/, conservative

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 2009, Business Insider, https://www.businessinsider.com/, center (American Owned Website)

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 1990, City Journal, https://www.city-journal.org/, far-right (Publisher is a Policy-Research Institute)

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 1961, Columbia Journalism Review, https://www.cjr.org/, center (Owned by Columbia University)

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 1896, Daily Mail, http://www.dailymail.co.uk/home/index.html, far-right (Fairly Common in England)

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 2005, HotAir,https://hotair.com/, center-right (All the founders are conservative)

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 2005, HuffPost, https://www.huffpost.com/, left-wing

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 1914, New Republic, https://newrepublic.com/, left (Magazine only comes out 10 times a year)

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 1998, Newsmax, https://www.newsmax.com/, right

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 2007, Politico, https://www.politico.com/, center-left

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 2012, Quartz, https://qz.com/, center (owned by Japanese Company)

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 2002, Google News, https://news.google.ca/, center (owned and operated by Google)

[the remark this answers is not in the document]
Comment (Carlos-Georgescu) [reply websites_list/news_websites.csv]: 1992, BBC, bbc.com, center (Headquarters in United Kingdom)

File: websites_list/news_websites.csv (hunk 2 above)
Comment (JmePRE) [inline websites_list/news_websites.csv]: 2002,Catholic News Agency,catholicnewsagency.com,Pro-Catholic,Conservative Right
2003,Xinhua News,xinhuanet.com,political,Pro-communist party
2004,Environment News Service,ens-newswire.com,liberal,pro_science
2005,Britain First,britainfirst.org,right_extremist
2006,Kaiser Health News,khn.org,least_biased
2007,Korea Central News Agency,kcna.kp,political,Pro-DPRK Government
2008,The Citizen (South Africa),citizen.co.za,right_center
2009,The New Humanitarian,thenewhumanitarian.org,left_center
2010,Egypt Today,egypttoday.com,right,political
2011,South China Morning Post,scmp.com,left_center
2012,Hong Kong Free Press,hongkongfp.com,political,Anti-chinese government
2013,Asian Correspondent,asiancorrespondent,com,left_center

File: websites_list/news_websites.csv [hunk 3]
```diff
@@ -1988,3 +1988,13 @@
 1986,New Indian Express,newindianexpress.com,Right Leaning
 1987,Dainik Bhaskar,bhaskar.com,Pro-NDA
```
Comment (IrDarcy) [inline websites_list/news_websites.csv]: 2009, [http://akeza.net/](http://akeza.net/), private news provider,Burundi

**Comment by Mayankstar364** [comment]
2001

**Pull request closed without merging**